src/__tests__/unit/lib/azure-importer/index.test.ts

```diff
@@ -1,52 +1,290 @@
 import {AzureImporterModel} from '../../../../lib/azure-importer';
+import {MetricsListOptionalParams} from '@azure/arm-monitor';
+jest.mock('@azure/identity', () => ({
+  __esModule: true,
+  DefaultAzureCredential: class MockAzureCredentials {},
+}));
+
+jest.mock('@azure/arm-monitor', () => ({
+  __esModule: true,
+
+  MonitorClient: class MockMonitorClient {
```

Comment:
@jmcook1186 can you please separate mocks to __mocks__/azure folder and import them ?